### PR TITLE
[Feature] Open bot to the webpack-cli

### PR DESCRIPTION
**Pull request by ematipico**
**What kind of change does this PR introduce?**
Feature

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Summary**
Fixes https://github.com/webpack/webpack-cli/issues/195

**Does this PR introduce a breaking change?**
No 

**Other information**
The configuration is copied from `webpack` repository and adapted to the current repository. The actions that the bot will do are:
- Adding labels `CI-ok` and `CI-not-ok` to the PRs **(labels needed)**
- Reporting error if linter and tests fail
- Reporting general error
- Adding label `PR: conflict` **(labels needed)**
- Adding labels `unreviewed`, `reviewed` and `review-outdated` to the PRs **(labels needed)**
- Adding label `PR: small` if the PR contains small changes **(label needed)**
- Adds a message if the PR is done from a branch that is master
- Moves tasks between repos of the same organization
- Marks inactive issues
- Checks issues every week

I also added two new npm scripts that will be used just by the bot.

### CODE REVIEW

**Comment by ematipico** [comment]
@ev1stensberg, @sokra New PR for the bot! Now it points to `v2` and the builds are not failing. 
Maybe as next step we could integrate codecov (the `v1` didn't have it so I preferred to not change the code again).

**Comment by ev1stensberg** [comment]
Notified Sokra about this, but the github hook comes up after checks succeed, so it is already integrated, yes? @sokra mind doing a quick review? This and one other issue left for a v2 RFC

**Comment by ev1stensberg** [comment]
Let's take this for a spin LGTM

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+      node_js: "stable"
+      env: JOB_PART=lint
+    - os: linux
+      node_js: "stable"
+      env: JOB_PART=integration
+    - os: linux
+      node_js: "8"
+      env: JOB_PART=integration
+    - os: linux
+      node_js: "7"
```
Comment (sokra) [inline .travis.yml]: 7 is a "in development" version number. Only even version numbers are node.js version you need to test.

Comment (ematipico) [reply .travis.yml]: The the versions will be stable, 8 and 6? As far as I understood webpack 4 won't support `v4` anymore.
Also, are the labels required for webpack-bot? I don't see it doing anything on new PRs.

**Review by supersezzie — APPROVED** [review]

**Pull request merged**